backend/SlothOrganizer/SlothOrganizer.Services/Auth/Tokens/AccessTokenService.cs

```diff
@@ -41,30 +70,12 @@ public string GetEmailFromToken(string token)
                 {
                     throw new Exception();
```

Comment:
What is the point to throw common exception, catch all exceptions and throw the new one?

Reply:
Fixed